app/services/CampaignService.scala

```diff
@@ -61,6 +61,34 @@ object CampaignService {
     }
   }
 
+  def getBenchmarksAcrossCampaigns(): Either[CampaignCentralApiError, Benchmarks] = {
```

Comment:
How come you use `Either`s and not `scala.util`'s `Success`/`Failure`?

Reply:
I typically avoid Try in Scala unless I'm using Java libraries that may throw exceptions. Even then I will convert from Try and model my exceptions using an ADT and typically return an Either. This has become much nicer now in 2.12 given Either is right biased.